server/models/index.ts

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+const config = require( '../config/config');
+const basename = path.basename(__filename);
+const db: any = {};
+
+let sequelize: Sequelize.Sequelize;
+
+if (process.env.CI){
```

Comment:
```
if ('false') {...}
```
Environment variables tend to be passed in as strings. You can lowercase and compare.